std-bits/table/src/main/java/org/enso/table/excel/ExcelRange.java

```diff
@@ -147,15 +148,66 @@ public static int parseA1Column(CharSequence column) throws IllegalArgumentExcep
     return parsed.value;
   }
 
+  /**
+   * Create an ExcelRange for a given set of columns.
+   *
+   * @param sheetName name of the containing sheet.
+   * @param leftColumn left-hand column index (1-based).
+   * @param rightColumn inclusive right-hand column index (1-based).
+   * @return ExcelRange for the given columns.
+   */
   public static ExcelRange forColumns(String sheetName, int leftColumn, int rightColumn) {
     return new ExcelRange(
         sheetName, leftColumn, WHOLE_ROW_OR_COLUMN, rightColumn, WHOLE_ROW_OR_COLUMN);
   }
 
+  /**
+   * Create an ExcelRange for a given set of rows.
+   *
+   * @param sheetName name of the containing sheet.
+   * @param topRow top row index (1-based).
+   * @param bottomRow inclusive bottom row index (1-based).
+   * @return ExcelRange for the given rows.
+   */
   public static ExcelRange forRows(String sheetName, int topRow, int bottomRow) {
     return new ExcelRange(sheetName, WHOLE_ROW_OR_COLUMN, topRow, WHOLE_ROW_OR_COLUMN, bottomRow);
   }
 
+  /**
+   * Given a single cell and the containing sheet, expand cover the connected table of cells.
```

Comment:
```suggestion
   * Given a single cell and the containing sheet, expand it to cover the connected table of cells.
```

Some words seem to be missing here. I tried inserting some that made sense, but I'm not exactly sure what was the original intent there.